assets/constants.ts

```diff
@@ -5845,6 +5845,13 @@ export default {
 
   "MAX_CURRENT_GOALS_COUNT": 5,
 
+  // The maximum number of questions can exceed the utmost 3
+  // (i.e., 18 questions) in some special cases when the user has attempted 14
+  // questions and another topic is tested for more accurate results, in
```

Comment:
Btw I think you should add a test demonstrating this behaviour (if you like, it's fine, in the test, to swap out the value of this constant for a smaller value like 5).

Reply:
Noted, I will add the tests in the diagnostic test engine service PR.